File: mobile/ios/Air/SubModules/UISettings/Settings/Models/SettingsSection.swift
//
//  SettingsSection.swift
//  UISettings
//
//  Created by Sina on 6/26/24.
//

import Foundation

struct SettingsSection: Identifiable {
    enum Section: String, CaseIterable, Equatable, Hashable {
        case header
        case accounts
        case general
        case walletData
        case questionAndAnswers
        case switchToCapacitor
        case signout
    }
    let id: Section
    var children: [SettingsItem]
}

File: mobile/android/air/SubModules/UISettings/src/main/java/org/mytonwallet/app_air/uisettings/viewControllers/settings/models/SettingsSection.kt
package org.mytonwallet.app_air.uisettings.viewControllers.settings.models

data class SettingsSection(
    val section: Section,
    var children: List<SettingsItem>
) {
    enum class Section {
        ACCOUNTS,
        WALLET_CONFIG,
        WALLET_DATA,
        NOT_IDENTIFIED
    }
}
